app/src/main/res/values/dimens.xml

```diff
@@ -12,5 +12,6 @@
     <dimen name="title_text_size">30sp</dimen>
     <dimen name="body_text_size">16sp</dimen>
     <dimen name="bullet_text_size">20sp</dimen>
+    <dimen name="title_margin_size">10sp</dimen>
```

Comment:
Only textSize should use sp. Here you have to use dp

Reply:
Done thx